codegen-sbt/src/main/scala/caliban/codegen/CompileTimeCalibanPlugin.scala

```diff
@@ -150,7 +150,13 @@ object CompileTimeCalibanServerPlugin extends AutoPlugin {
 
   override lazy val projectSettings: Seq[Def.Setting[_]] =
     Seq(
-      libraryDependencies += "com.github.ghostdogpr" %% "caliban-tools" % caliban.codegen.BuildInfo.version % Compile,
+      libraryDependencies += {
+        CrossVersion.partialVersion((thisProject / scalaVersion).value) match {
```

Comment:
It didn't pick up the version properly?

Reply:
It's using the Scala version of sbt apparently. At runtime, our tests were crashing. We'll add some tests to prove that it working now.